src/document.c

```diff
@@ -825,7 +825,8 @@ GeanyDocument *document_new_file_if_non_open(void)
 }
 
 
-/**
+/** @girskip
```

Comment:
Mmmkay I see.  No new opinion just yet, but I was thinking: new proposed name `document_create_file()` suggests it *creates* a file, literally; which ain't the case, it just opens a new buffer not baked to a file (so, quite the contrary).  `new` had less attached meaning to it.